std/complex.d

```diff
@@ -887,6 +885,31 @@ Complex!real expi(real y)  @trusted pure nothrow @nogc
 
 
 /**
+    Params: y = A real number.
+    Returns: The value of cosh(y) + i sinh(y)
+
+    Note:
+    $(D coshisinh) is included here for convenience and for easy migration of code
+    that uses $(REF _coshisinh, std,math).
+*/
+Complex!real coshisinh(real y) @safe pure nothrow @nogc
+{
+    import std.math : cosh, sinh;
+    return Complex!real(cosh(y), sinh(y));
```

Comment:
Is this the most efficient way of performing the calculation? I'm only asking because the one in `std.math` is implemented differently, using `exp()`/`expm1()`.